Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -159,6 +159,30 @@ public void Terminate(IdentityAddress activeIdentity, DeviceId activeDevice)
         AuditLog.Add(auditLogEntry);
     }
 
+    public void Reactivate(IdentityAddress activeIdentity, DeviceId activeDevice)
```

Comment:
This method should be called `RequestReactivation`, as it only requests the reactivation. To complete the reactivation an approval is required.

Reply:
Changes made.